.gitignore

```diff
@@ -1,5 +1,3 @@
 node_modules/
-test/data/output/*
```

Comment:
right now currently I ignore the test file directories and just check the summary files since those are a bit easier to parse through and should match with the end files.